apps/ubuntu_bootstrap/lib/app/installation_step.dart

```diff
@@ -41,6 +43,8 @@ enum InstallationStep with RouteName {
   refresh(RefreshPage.new, allowedToHide: true),
   tryOrInstall(TryOrInstallPage.new, discreteStep: false, allowedToHide: true),
   autoinstall(AutoinstallPage.new, allowedToHide: true),
+  autoinstallLandscapeDomain(LandscapeDomainPage.new),
+  landscape(LandscapePage.new, required: true),
```

Comment:
Can we make all the enums and page names consistent (`AutoinstallLandscape...`)?